azure-iot-hub/tests/test_digital_twin_client.py

```diff
@@ -7,8 +7,8 @@
 import pytest
 from azure.iot.hub.protocol.models import AuthenticationMechanism
 from azure.iot.hub.digital_twin_client import DigitalTwinClient
-from azure.iot.hub.auth import ConnectionStringAuthentication
-from azure.iot.hub.protocol.iot_hub_gateway_service_ap_is import IotHubGatewayServiceAPIs
+import azure.iot.hub.auth
+import azure.iot.hub.protocol.iot_hub_gateway_service_ap_is
```

Comment:
It's probably best to import objects directly in test modules (as opposed to implementations) in order to make tests a bit more readable, especially since that's how other objects in this test file already work.

Unlike with the implementation, there's no benefit to doing one over the other in temrs of mocking, so just do something consistent.

And certainly don't import a full path like this, if you were really tied to importing the module for whatever reason you'd do

```python
from azure.iot.hub import auth
from azure.iot.hub.protocol import iot_hub_gateway_service_ap_is
```

so that you don't have to write out the full module location every time.